### PR TITLE
Add issue templates to the repo

### PR DESCRIPTION
**Pull request by ericwb**
Adds the following:
* template for bugs, auto-labeled with bug label
* template for features, auto-labeled with enhancement label
* link to the discussions for questions
* emojis!

Signed-off-by: Eric Brown <browne@vmware.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: ❓ Ask a question
+    url: https://github.com/vmware-labs/ml-conversational-analytic-tool/discussions
```
Comment (difince) [inline .github/ISSUE_TEMPLATE/config.yml:4]: Thank you @ericwb 
LGTM, except that this link returns 404. I am  not sure if this is ok at that stage, but wanted no mention it 

I did a research and I found that a GitHub feature - `GitHub discussions` could be easily enabled for the project 
https://docs.github.com/en/discussions

Comment (ericwb) [reply .github/ISSUE_TEMPLATE/config.yml:4]: Hmm, weird, the link works for me: https://github.com/vmware-labs/ml-conversational-analytic-tool/discussions. Can you try again?

Comment (annajung) [reply .github/ISSUE_TEMPLATE/config.yml:4]: the link also works for me

**Review by annajung — APPROVED** [review]
lgtm thank you!

**Pull request merged**